app/app.go

```diff
@@ -536,11 +559,60 @@ func New(
 	// CanWithdrawInvariant invariant.
 	// NOTE: staking module is required if HistoricalEntries param > 0
 	app.mm.SetOrderBeginBlockers(
-		upgradetypes.ModuleName, capabilitytypes.ModuleName, minttypes.ModuleName, distrtypes.ModuleName, slashingtypes.ModuleName,
-		evidencetypes.ModuleName, stakingtypes.ModuleName, ibchost.ModuleName,
+		upgradetypes.ModuleName,
+		capabilitytypes.ModuleName,
+		minttypes.ModuleName,
+		distrtypes.ModuleName,
+		slashingtypes.ModuleName,
+		evidencetypes.ModuleName,
+		stakingtypes.ModuleName,
+		ibchost.ModuleName,
+		feegrant.ModuleName,
+		datapooltypes.ModuleName,
+		oracletypes.ModuleName,
+		vestingtypes.ModuleName,
+		govtypes.ModuleName,
+		paramstypes.ModuleName,
+		ibctransfertypes.ModuleName,
+		datadealtypes.ModuleName,
+		authtypes.ModuleName,
+		aoltypes.ModuleName,
+		didtypes.ModuleName,
+		wasm.ModuleName,
+		banktypes.ModuleName,
+		crisistypes.ModuleName,
+		burntypes.ModuleName,
+		genutiltypes.ModuleName,
+		authz.ModuleName,
```

Comment:
I have a question. In this part, because the `datatdeal` module doesn't have the `BeginBlocker` and `EndBlocker`, so you didn't add `datadeal`. Is it right?

Reply:
Actually not. As I mentioned, all modules should be included in all the SetOrder{BeginBlockers, EndBlockers, InitGenesis} funcs. `datadeal` is added in line 577 :)

Reply:
Oh, I missed it 😂 Thanks for comments!